plugin/trino-singlestore/src/test/java/io/trino/plugin/singlestore/TestingSingleStoreServer.java

```diff
@@ -31,7 +31,8 @@
 public class TestingSingleStoreServer
         extends JdbcDatabaseContainer<TestingSingleStoreServer>
 {
-    public static final String DEFAULT_VERSION = "7.8";
+    // https://docs.singlestore.com/db/v9.0/support/singlestore-software-end-of-life-eol-policy
+    public static final String DEFAULT_VERSION = "8.7";
```

Comment:
Could you update the requirement in singlestore.md? 

https://github.com/trinodb/trino/blob/fa17547bb4bb2a1041bb0a1d08e6773dd645893c/docs/src/main/sphinx/connector/singlestore.md?plain=1#L20